velox/functions/sparksql/aggregates/BloomFilterAggAggregate.cpp

```diff
@@ -56,7 +56,7 @@ struct BloomFilterAccumulator {
   }
 
   BloomFilter<StlAllocator<uint64_t>> bloomFilter;
-}; // namespace
+};
```

Comment:
unintentional change?

Reply:
Looks like the comment was in wrong place. It's not the boundary of the anonymous namespace.

Reply:
Double checked, you are right.